_data/speakers.yml

```diff
@@ -1,162 +1,147 @@
   thumbnailUrl: joao-barros.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 013
+- id: 013
   name: "Luís Filipe"
   surname: "Gonçalves"
   company: "Jumpseller"
   title: "Software Engineer and Co-Founder"
   bio: ""
   thumbnailUrl: filipe-goncalves.jpeg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 014
+- id: 014
   name: "Marco"
   surname: "Pinto"
   company: "Fabamaq"
   title: "Game developer"
   bio: ""
   thumbnailUrl: marco-pinto.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 015
+- id: 015
   name: "Joss"
   surname: "Santos"
   company: "Jscrambler"
   title: "Product Growth Team Lead"
   bio: ""
   thumbnailUrl: joss-santos.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 016
+- id: 016
   name: "Emanuel"
   surname: "Fernandes"
   company: "Jscrambler"
   title: ""
   bio: ""
   thumbnailUrl: emanuel-fernandes.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 017
+- id: 017
   name: "Cátia"
   surname: "Domingues"
   company: "Glintt"
   title: "Talent Acquisition & Employer Branding"
   bio: ""
   thumbnailUrl: catia-domingues.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 018
+- id: 018
   name: "Stéphanie"
   surname: "Dermagne"
   company: "IT-Sector"
   title: "Talent Acquisition Manager"
   bio: ""
   thumbnailUrl: stephanie-dermagne.JPEG
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 019
+- id: 019
   name: "André"
   surname: "Lago"
   company: "Google"
   title: "Software Engineer and Tech Lead"
   bio: ""
   thumbnailUrl: andre-lago.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 020
+- id: 020
   name: "Rui"
   surname: "Vilaça"
   company: "Critical Manufacturing"
   title: "Senior Software Engineer"
   bio: ""
   thumbnailUrl: rui-vilaca.jpg
   rockstar: true
-  social: 
+  social:
     #- {name: "google-plus", link: "https://plus.google.com/+DawidOstrowski/"}
-- 
-  id: 021
-  name: ""
-  surname: ""
+- id: 021
+  name: "Filipa"
+  surname: "Muchacho"
```

Comment:
Não tenho a certeza que sera oradora, a partida e so networking, ainda nao confirmou

Reply:
Em princípio podemos mudar mais tarde se não for?